department-of-veterans-affairs/caseflow
# Kamalam7/appeals 29209

Resolves https://jira.devops.va.gov/browse/APPEALS-29209

# Description
If the Package document type is 10182, in the CMP info table we need to show the Intake appeal button on the Review package page. And if the package document type is other than 10182, the Intake appeal button should disappear.

## Acceptance Criteria
- [x] Mail user will find an "Intake Appeal" button on the Review Package page
       "Intake Appeal" button is only shown to the mail user if the Package Document Type in the CMP Information 
        section is "10182"
        If the package document type is not a "10182" the button will not appear

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below

1. Login as a Mail user( Jolly_Postman) and switch user.
2. In the App selector, click on Queue.
3. In Queue, Your cases page, Click on Switch views and select Correspondence cases.
<img width="828" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/32e22adf-9e64-460f-8135-753f3663952e">
4. In the Correspondence cases, click on Veteran details. This will take to the Review package page.
<img width="827" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/a4b15d2c-bac9-442c-87be-afdcaf08a283">
<img width="887" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/3e947633-1d4f-4fee-9148-d345ed946126">
5. In the Review package page, under CMP Information Table, if the Package document type is 10182, we need to show the Intake appeal button.
<img width="799" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/aec6260a-c13c-4c63-b233-65023e19c9c9">
<img width="849" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/d95d858e-1362-460d-a1e3-2b93300429c2">
6. If the Package document type is other than 10182, Intake appeal button should disappear.
<img width="801" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/a51addea-2bee-4dbe-b2e9-a544baea6d59">
<img width="849" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110078646/5afef5d1-0366-4eaa-b3fc-0dfd26b4c05c">



# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
